packages/paste-website/src/components/site-wrapper/sidebar/SidebarSeparator.tsx

```diff
@@ -1,4 +1,3 @@
-import * as React from 'react';
```

Comment:
question: why this change? Usually if a file contains JSX it's common practice to import React even if isn't actively used for certain build tools / bundling purposes. Just curious what error you were seeing to remove these and from where

Reply:
tsx check ignored here: https://github.com/twilio-labs/paste/pull/2945